internal/display/organizations.go

```diff
@@ -44,7 +44,7 @@ func (v *organizationView) Object() interface{} {
 	return v.raw
 }
 
-func (r *Renderer) OrganizationList(organizations []*management.Organization) {
+func (r *Renderer) OrganizationList(organizations []management.Organization) {
```

Comment:
Is there a particular reason we're removing the pointer here? 🤔

Reply:
You're right, this is not needed. I reverted that change.